users/README.md

```diff
@@ -265,3 +266,36 @@ Updates data of the User.
   - **Code:** 503
     - **Content:**
       `{ 'statusCode': 503, 'error': 'Service Unavailable', 'message': 'Something went wrong please contact admin' }`
+
+## **PATCH /users/:id/roles**
+
+Updates roles for the User.
+
+- **Params**  
+  None
+- **Query**  
+  None
+- **Headers**  
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
+- **Body**
+  `{
+    rolename: <boolean>
+  }`
+- **Success Response:**
+  - **Code:** 204
+    - **Content:** `{ 'message': 'role updated successfully!'}`
```

Comment:
Along with message we should also return the updated data. Returning the updated data can be useful for clients that need to display the updated information to the user.

Reply:
Its wrong success status will be 200

Reply:
Success status 200 is fine. what I meant was along with the response you could also return the updated role values. Its optional but provide more value to the one consuming this API